content/blog/2023/announcing-devsprints.md

```diff
@@ -0,0 +1,47 @@
+Title: Join the Fun: Participate in DevSprints at PyCon India 2023
+Date: 2023-09-19 16:00
+Category: 2023
+Slug: devsprints
+Summary: Make your first contribution to open source at PyCon India 2023 DevSprints
+
+PyCon India 2023 is the perfect place to get started with contributing to open-source projects. We are organizing DevSprints on 3rd October 2023.
+
+## What are DevSprints?
+
+[Devsprints](https://in.pycon.org/cfp/devsprint-2023/proposals/) are a whole-day event organized at PyCon India 2023 to promote open-source contribution and bring more people into the open-source community.
+
+**Are you interested in making your first open-source contribution? Or are you a seasoned contributor looking to help out others? Then Devsprints is the place for you!** At Devsprints, active contributors of open-source projects are there to guide you through the process of making your very first open-source contribution. Right from setting up the build environments, to developing and testing code, to submitting patches for bugs and features. If you are stuck anywhere, the mentors and your fellow contributors will be right there to help you figure out issues and resolve them as best as they can! Many participants have started their open-source journey at PyCon India DevSprints and have gone on to become regular contributors to various open-source projects.
+
+## Who can participate?
+
+***You!*** Whether you're a seasoned contributor or just starting, the DevSprints at PyCon India 2023 are the perfect opportunity to get to work. Our mentors will be on hand to provide instant help whenever you need it, so you can keep moving forward. It is a great opportunity to get accustomed to the ways of the open-source and contribute to the projects you love.
+
+> `What good is a tech conference without getting your hands dirty writing some code?`
+
+## When and Where?
+
+**After the conference, we will be organizing DevSprints on 3rd October 2023. The DevSprints will be held at [Phenom People Pvt Ltd, Whitefield, Kondapur.](https://goo.gl/maps/hmHBapTF6bMNJWWx5)**
+
+The timings for the DevSprints will be from 10:00 AM to 5:00 PM.
+
+### How to participate?
+
+You do not need any additional tickets to participate in the DevSprints. If you have purchased a conference ticket, you can attend the DevSprints as well. However the capacity of the venue is limited, so we will be sharing a form to register for the DevSprints.
```

Comment:
Any person with a Valid Conference ticket is free to attend Devsprints, subject to the maximum capacity of 200 participants. Entry will be permitted on a first come first serve basis at the Venue.

Reply:
we're not sharing a form?

Reply:
```suggestion
You do not need any additional tickets to participate in the DevSprints. Any person with a Valid Conference ticket is free to attend Devsprints. However, the capacity of the venue is limited, up to a maximum of 200 participants. Entry will be permitted on a first come first serve basis at the Venue.
```

Reply:
Nope no form this time.